python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -842,15 +842,6 @@ def num_idle_actors(self) -> int:
             for running_actor in self._running_actors.values()
         )
 
-    def num_free_slots(self) -> int:
-        """Return the number of free slots for task execution."""
-        if not self._running_actors:
-            return 0
-        return sum(
-            max(0, self._max_tasks_in_flight - running_actor.num_tasks_in_flight)
-            for running_actor in self._running_actors.values()
-        )
-
```

Comment:
This'll cause a slight change in behavior. Previously, the `ActorPoolMapOperato` would only receive inputs if there are free slots on running actors; now, it'll accept inputs if there are free slots in running or pending actors.

Reply:
I see. Should we also exclude pending actors in the base class `ActorPoolMapOperator.num_free_task_slots`?